### ISSUE TITLE
can not reimplement the reported result, can you help me?

### ISSUE THREAD

**Issue by HanChenh** [YingqianWang/OACC-Net#6]
i try to train the model on a RTX 3090 with following hyper-parameters around 1000 epochs, it turns out that the MSE metric cannot go up higher, is this a normal phenomenon? 
![img_v2_3e1ab697-bf68-4fe2-8aee-006361597d1g](https://github.com/YingqianWang/OACC-Net/assets/139227917/58a13de0-8657-45f0-8478-72c2ce5b9a3b)
![img_v2_68dc8126-db45-4334-aefb-ecd7b52213ag](https://github.com/YingqianWang/OACC-Net/assets/139227917/e68c236c-ce63-494c-90a1-c7b1bf2a9544)

**Comment by syt06007** [comment]
Hello, I'm encountering the same issue. Could you kindly share how you managed to resolve it?
